meteoroid/core/tests/libs/clients/test_open_whisk_client.py

```diff
@@ -0,0 +1,177 @@
+import unittest
+from unittest.mock import Mock
+from unittest.mock import patch
+
+from ....libs.clients.open_whisk_client import OpenWhiskClient
+from ....libs.clients.open_whisk_client import OpenWhiskClientException
+
+
+class MockResponse(object):
```

Comment:
the class doesn't use?

Reply:
Thank you for your comment.
MockResponse is not used. It has been deleted.